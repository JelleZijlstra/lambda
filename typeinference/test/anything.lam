match 3 with {
	_ -> print 3
}
