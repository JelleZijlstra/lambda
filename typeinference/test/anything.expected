%%stdout%%
3
Result: module
	let __result__ = ()
end
%%stderr%%
